SwiftyJSONAccelerator/SJEditorViewController.swift

```diff
@@ -110,6 +121,30 @@ class SJEditorViewController: NSViewController, NSTextViewDelegate {
         }
     }
 
+    @IBAction func recalcEnabledBoxes(sender: AnyObject) {
+        
+        let supportSwiftyState = self.supportSwiftyJSONCheckbox?.state == 1 ? true : false
+        let supportObjectMapperState = self.supportObjectMapperCheckbox?.state == 1 ? true : false
+
+        if supportSwiftyState
+        {
```

Comment:
Sure. I just pushed one that fixed that and the other issues. I think I got all of them.